src_firmware/src/shell.c

```diff
@@ -56,21 +54,17 @@ static int cmd_speed(const struct shell *shell, size_t argc, char *argv[])
 		if (ret == SUCCESS) {
```

Comment:
Note: worth to create a `log_error()` function which would map common errors and print them to avoid lots of if else statements. Can be done later

Reply:
added todo